src/tutorial/microbe_editor/ChemoreceptorPlacementTutorial.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Tutorial
+{
+    public class ChemoreceptorPlacementTutorial : EditorEntryCountingTutorial
+    {
+        public ChemoreceptorPlacementTutorial()
+        {
+            CanTrigger = false;
+        }
+
+        public override string ClosedByName => "ChemoreceptorPlacementTutorial";
+
+        protected override int TriggersOnNthEditorSession => 3;
```

Comment:
Does this work nicely now with the stay small tutorial? If not I think just incrementing this value by one would be a pretty okay thing to try.  I thought about making this tutorial not appear if a chemoreceptor is already placed, but I think this tutorial could serve as the tutorial telling people about how to edit an organelle, so this is still useful to triggere even if a chemoreceptor is already placed.

Reply:
So should this tutorial tell the player that the chemoreceptor exists or should it tell the player how to edit an organelle?

Reply:
I think it can do all of those?

> The chemoreceptor can be used to easily find compounds that are farther away.

> After placing the chemoreceptor it needs to be modified to select what it detects. Some other organelles can also be modified after placing them to alter their functionality.

I think that amount of text should fit in one tutorial box. Maybe it also needs a bit of text saying to right click the thing (though this should use the keybinding bbcode to show the actual keybinding).

Reply:
Right, in that case this triggers even if there is no chemoreceptor in the cell, so there would be no way to close this when the organelle is modified like you requested in your other comment

Reply:
But the tutorial could wait for the player to place a chemoreceptor (or any other modifiable organelle) and close automatically when the modify option is triggered. For people who don't want to place an organelle we should leave the X close button active on this tutorial in case someone wants to skip it.